### DISCUSSION TITLE
Improving accessibility guidance

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/wai-mission#8] [Specific, miscellaneous]
WCAG 3 is years away from completion, even if we figure out the major issues with testing and conformance.

What can we do to improve accessibility guidance in the meantime?

1. Big picture &mdash; Should we take fresh look at re-imagining accessibility guidance for the next 5 years? What can we leverage and learn from [Silver/WCAG 3](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/) work, [Tutorials](https://www.w3.org/WAI/tutorials/), [COGA Supplement Guidance](https://www.w3.org/WAI/WCAG2/supplemental/about/), [What's New persona quotes and 'In Brief's](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#257-dragging-movements-aa), etc.?

2. Are there some relatively quick and easy things we can do to make improvements sooner?
One little yet impactful thing we are working on is adding "In brief" summaries and persona quotes, e.g., [Dragging in What's New in 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#257-dragging-movements-aa). We are adding "In Brief" summaries to all of the Understanding WCAG documents.
  - Should we prioritize editing Understanding docs even more? e.g., perhaps surfacing sufficient Techniques would help (e.g., seems like it would for [3.3.8](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication))
  - Should we prioritize developing additional [Supplemental Guidance](https://www.w3.org/WAI/WCAG2/supplemental/about/), e.g., to cover the low vision, mobile, and I18N issues that didn't make it into WCAG 2?
  - Should we try to recruit technical editing resources?
  - Other?
3. Are ideas in-between to consider?

**Comment by mitchellevan** [comment]
> Should we try to recruit technical editing resources?

Yes. Good writing quality supports consumers of the specs through reputation, usability, and inclusion. Sufficient quantity of skilled editors encourages more participation by technology experts with different strengths.
